social-network/src/Article.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Article extends Component {
+    render() {
+        const datePosted = this.props.article.postedDate;
+
+        function formatDate(date) {
+			const months = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'];
+			const newDate = new Date(datePosted);
+            const day = newDate.getDate();
+            const month = months[newDate.getMonth()];
+
+            return <span>{month} {day}</span>
```

Comment:
It can be confusing to set up, but the date object can format strings for you: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString